README.md

```diff
@@ -1,2 +1,170 @@
-# blog_app-
-This is an online application that enables users to compose, modify, and publish blog entries is called a Ruby blog app. Usually, the Ruby on Rails framework—a well-liked option for creating web applications—is used to build it.
+<a name="readme-top"></a>
+
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
ok i will do that thany for all you feedbacks